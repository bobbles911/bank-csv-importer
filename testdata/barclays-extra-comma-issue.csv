Number,Date,Account,Amount,Subcategory,Memo
0,01/04/2022,11-11-11 12345678,-11.50,Credit Payment,AAAAAA AAAAAA AAAAAA       	1 @ £1.11	
	0,04/05/2021,11-11-11 12345678,-1111.00,Bill Payment,AAAAAA AAAAAA AAAAAA    	AAAAAA-001 AAA	
	0,04/05/2021,11-11-11 12345678,-2222.00,Funds Transfer,12234 23532525       	AAAAAA AA	
	0,04/05/2021,11-11-11 12345678,14124.00,Counter Credit,AAAAAA AAAAAA          	AAAAAAAAAAAA AA
	0,04/05/2021,11-11-11 12345678,23.66,Counter Credit,AAAAAA A B             	AAA, AAAAAA AAAAAA. AAA
	0,30/04/2021,11-11-11 12345678,534535.55,Transfer,AAAAAA AAA *         	1321321*AAAAAA Aa * AA
	0,30/04/2021,11-11-11 12345678,2423.33,Transfer,AAAAAA AAA *         	23423423*AAAAAA Aa * AA

