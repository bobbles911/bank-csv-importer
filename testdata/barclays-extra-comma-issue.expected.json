{
	"entireColumnTypes" : [null, "Date", null, "Number", null, null, null],
	"headerGuesses" : {
		"date" : 1,
		"amount" : 3,
		"balance" : null,
		"description" : 5
	}
}
